woocommerce/woocommerce-ios
# Product Variations: Add Attribute Options screen (WIP)

## Description
As part of #3182, in this PR I implemented the basic components for managing the components of the Attribute Options screen. For the moment, the data showed inside are not related for to real options, and it's not possible to do any action. The action and the data (that should be fetched using an API) will be part of other PRs.

The functionality it's under the M5 feature flag, so it will be not visible to our users, since it's still a WIP.

## Testing
1. Open a Variable Product without variations.
2. Tap the cell "Add variations".
3. The screen "Add Attribute" should be presented.
4. In the "Add Attribute" screen you should see the list of global attributes + local product attributes.
5a. Tap an existing attribute. The new screen should be opened.
5b. Try to write a new attribute in the text field and tap Next. The new screen should be opened.

## Screenshot
<img src="https://user-images.githubusercontent.com/495617/102634537-42df8280-4152-11eb-97ad-26d404382673.png" width=300 />


Update release notes:
